bin/run.sh

```diff
@@ -36,19 +36,40 @@ tmp_dir=$(mktemp -d -t "exercism-verify-${slug}-XXXXX")
 trap 'rm -rf "$tmp_dir"' EXIT
 cp -r "${solution_dir}/." "${tmp_dir}"
 cd "${tmp_dir}"
-
 jq -r '.files.test[]' .meta/config.json | while read -r test_file; do
     sed -i -E 's/(test|it).skip/\1/g' "${test_file}"
 done
 
+# Capture the output of the test run in case of failure or error
 test_output=$(arturo tester.art 2>&1)
 
-# Write the results.json file based on the exit code of the command that was 
-# just executed that tested the implementation file
-if [ $? -eq 0 ]; then
-    jq -n '{version: 1, status: "pass"}' > ${results_file}
-else
+# Tweak the generated test file structure to valid Arturo code 
+sed -i \
```

Comment:
If you use `sed -i` this won't work on MacOS. At that point, you might want to just switch to using `bin/bash`